src/pcapi/alembic/versions/3bf0e437edb0_add_expiration_date_on_deposit.py

```diff
@@ -0,0 +1,29 @@
+"""add_expiration_date_on_deposit
+
+Revision ID: 3bf0e437edb0
+Revises: 85adf538245f
+Create Date: 2021-02-08 09:07:14.217608
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "3bf0e437edb0"
+down_revision = "85adf538245f"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("deposit", sa.Column("expirationDate", sa.DateTime(), nullable=True))
+    op.execute('UPDATE deposit SET "expirationDate" = "dateCreated" + interval \'2 years\'')
```

Comment:
```suggestion
    op.execute("""UPDATE deposit SET "expirationDate" = "dateCreated" + interval '2 years'""")
```
pour éviter d'avoir à échapper les apostrophes.